github-for-jira.sd.yml

```diff
@@ -550,3 +550,88 @@ environmentOverrides:
           ip_whitelist:
             - public
           routes: *blackholeSpammingIPs
+
+      - type: sqs
+        name: backfill
+        alarms:
+          AlarmOnTooManyMessages:
+            MetricName: ApproximateNumberOfMessagesVisible
+            Namespace: AWS/SQS
+            Description: "Message count for backfill queue too high! Follow the runbook: https://hello.atlassian.net/wiki/spaces/PF/pages/1550488861/HOWTO+Respond+to+Too+Many+Messages+on+the+Queue+Alert"
+            Threshold: 30 #~ Autoscaling Threshold x 2. So we'll be alerted if autoscaling failed to fix the piling messages
+            Priority: High
+            Dimensions:
+              - Name: QueueName
+                Value: { "Ref": "QueueName" }
+            EvaluationPeriods: 5
```

Comment:
Set evaluation period to 5, so we'll check 5 times with 5 minutes delay (Which mean we'll get an alert if the situation on the queue will stay for 20 minutes)